### ISSUE TITLE
 /usr/share/sbt/bin/sbt-launch-lib.bash: line 58:   405 Killed  "$@" | subprocess installed post-installation script returned error exit status 137

### ISSUE THREAD

**Issue by sfergusonvolute** [sbt/sbt#3414]
## steps

docker build --no-cache -t docker-play .

Dockerfile:

```
###########################################################
# Dockerfile to build PLAY Installed Containers
# Based on Ubuntu
############################################################

# Set the base image to Ubuntu
FROM ubuntu:latest

# File Author / Maintainer
MAINTAINER sfergusonvolute

# RUN apt-get install -y apt-transport-https

# Update the apt repository
RUN apt-get update

# Install necessary tools
RUN apt-get install -y apt-transport-https
RUN apt-get install -y apt-utils
RUN apt-get install -y docker
RUN apt-get install -y vim
RUN apt-get install -y ssh
RUN apt-get install -y dialog
RUN apt-get install -y net-tools

# Add sources
RUN echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
RUN apt-get update

# Upgrade the distro
#RUN apt-get dist-upgrade -y

# Install the requirements for the application
RUN apt-get install openjdk-8-jdk-headless -y
RUN apt-get install sbt -y
```

## problem
I had no problems using docker to build my containers on Monday, but today I receive this error:

```
[warn] No sbt.version set in project/build.properties, base directory: /
/usr/share/sbt/bin/sbt-launch-lib.bash: line 58:   405 Killed                  "$@"
dpkg: error processing package sbt (--configure):
 subprocess installed post-installation script returned error exit status 137
Processing triggers for libc-bin (2.23-0ubuntu9) ...
Processing triggers for systemd (229-4ubuntu19) ...
Errors were encountered while processing:
 sbt
E: Sub-process /usr/bin/dpkg returned an error code (1)
The command '/bin/sh -c apt-get install sbt -y' returned a non-zero code: 100
```

## expectation
A successful unattended installation of sbt.

## notes

sbt version: 0.13.16

**Comment by sfergusonvolute** [comment]
In my Docker builds, I was able to do a work around to pull and install certain older versions of .deb files using a different technique, but whenever I get to the sbt update step of my process I still get an error.

```
RUN curl -L -o sbt.deb http://dl.bintray.com/sbt/debian/sbt-0.13.11.deb
RUN dpkg -i sbt.deb

	[SUCCESSFUL ] org.fusesource.jansi#jansi;1.4!jansi.jar (18ms)
:: retrieving :: org.scala-sbt#boot-scala
	confs: [default]
	5 artifacts copied, 0 already retrieved (24494kB/2791ms)
/usr/share/sbt-launcher-packaging/bin/sbt-launch-lib.bash: line 41:    64 Killed                  "$@"
The command '/bin/sh -c sbt update' returned a non-zero code: 137
```

Thankfully, my existing stand alone servers are not having issues when I run an sbt update. I am only having this issue with new builds/installs. (Same application code set.)

Take care,
Sarah Ann Ferguson

**Comment by eed3si9n** [comment]
This was fixed in https://github.com/sbt/sbt-launcher-package/pull/177

**Comment by milnomada** [comment]
I solved this issue I had with `sbt-1.1.5.deb` version by:
```
sudo dpkg -r sbt
sudo dpkg --purge sbt
wget https://dl.bintray.com/sbt/debian/sbt-0.13.0.deb
sudo dpkg -i sbt-0.13.0.deb
```

Then, as I'm using java 8, I had to patch the following line so a memory limitation was applied during assembly:
in `sbt/bin/sbt-launch-lib.bash` at line 90:

```
#echo "-Xms${mem}m -Xmx${mem}m -XX:MaxPermSize=${perm}m -XX:ReservedCodeCacheSize=${codecache}m"
echo "-Xms${mem}m -Xmx${mem}m -XX:MaxMetaspaceSize=${perm}m -XX:ReservedCodeCacheSize=${codecache}m"
```

As a *note*, watching memory consumption in htop, it was lower with `0.13.0` version.

**Comment by SwapnilBorse123** [comment]
@milnomada I tried doing what you have done. Unfortunately, it doesn't solve my issue. When I try to do a sbt compile for my play application on a EC2 console, I still keep getting the following error.

/usr/share/sbt/bin/sbt-launch-lib.bash: line 46:  5523 Killed                  "$@"

Any help would be much much appreciated. I have invested more than a day in this and now it is kinda nerve breaking.
